.github/workflows/coding.yml

```
@@ -28,8 +28,9 @@
       #  run: npm run lint  # Assuming you have a lint script in your package.json  
   
   # TEST
-  test-scala-coding:  
+  test-scala:
     runs-on: ubuntu-latest  
```

YAMLlint (1.35.1)

[error] trailing spaces

(trailing-spaces)